test/expected/merge.out

```diff
@@ -1,46 +1,48 @@
 -- This file and its contents are licensed under the Apache License 2.0.
 -- Please see the included NOTICE for copyright information and
 -- LICENSE-APACHE for a copy of the license.
--- Create conditions table with location and temperature
-CREATE TABLE conditions (
+\c :TEST_DBNAME :ROLE_SUPERUSER
```

Comment:
What about testing the MERGE commands without a SUPERUSER?? Just because on cloud we do not give a SUPERUSER for our customers.